databricks/koalas/series.py

```diff
@@ -1085,6 +1085,143 @@ def corr(self, other, method='pearson'):
         c = df.corr(method=method)
         return c.loc["corr_arg1", "corr_arg2"]
 
+    def nsmallest(self, n: int = 5, keep: str = 'first') -> 'Series':
+        """
+        Return the smallest `n` elements.
+
+        Parameters
+        ----------
+        n : int, default 5
+            Return this many ascending sorted values.
+        keep : {'first', 'last', 'all'}, default 'first'
+            When there are duplicate values that cannot all fit in a
+            Series of `n` elements:
+            - ``first`` : take the first occurrences based on the index order
+            - ``last`` : take the last occurrences based on the index order
+            - ``all`` : keep all occurrences. This can result in a Series of
+                size larger than `n`.
+
+        Please note the Koalas: default behaviour is like "all"
```

Comment:
Is` keep`'s default first or all?

Reply:
Keep in Pandas defaults to first. But we don't have a way to replicate that here so Koalas behaves like all